compiler-rt/lib/plsan/plsan.cpp

```diff
@@ -103,11 +106,9 @@ extern "C" void *__plsan_memmove(void *dest, void *src, size_t num) {
 
 namespace __plsan {
 
-Plsan::Plsan() {
-  handler = (PlsanHandler *)__sanitizer::InternalAlloc(sizeof(PlsanHandler));
-}
```

Comment:
And this should have been removed in the first commit of this pull request because Plsan Handler is removed there

Reply:
hmm. okay